python/lsst/ts/observatory/control/maintel/comcam.py

```diff
@@ -240,7 +241,43 @@ async def setup_filter(self, filter):
         if filter is None:
```

Comment:
Do you think this method will be most commonly called with `filter = None`? The reason I ask is that, if the most common call is to provide a filter, it may be better to do;

```
if fiter is not None:
   ...
```

Reply:
I think flipping the check is better since it pulls the big block of functionality up. Did look kind of weird on retrospect to see the simple "just return" line first.